.github/workflows/smoke-test-pr-merge.yml

```diff
@@ -139,27 +139,35 @@ jobs:
                   test-type: E2E
 
     k6:
-        name: Run k6 Performance tests
-        needs: [api]
+        name: Runs k6 Performance tests
+        if: (github.event.pull_request.user.login != 'github-actions[bot]') && (github.event.pull_request.merged == true) && (github.event.pull_request.base.ref == 'trunk')
         runs-on: ubuntu-20.04
         permissions:
             contents: read
         steps:
             - uses: actions/checkout@v3
-              with:
-                  ref: ${{ github.event.pull_request.merge_commit_sha }}
 
             - name: Setup WooCommerce Monorepo
               uses: ./.github/actions/setup-woocommerce-monorepo
-
-            - name: Setup local test environment
-              uses: ./.github/actions/tests/setup-local-test-environment
               with:
-                  test-type: k6
+                  install: '@woocommerce/plugin-woocommerce...'
+                  build: '@woocommerce/plugin-woocommerce'
 
-            - name: Run k6 performance tests
-              id: run-k6-composite-action
-              uses: './.github/actions/tests/run-k6-tests'
+            - name: Load docker images and start containers.
+              working-directory: plugins/woocommerce
+              env:
+                  ENABLE_HPOS: 0
```

Comment:
We might want to enable HPOS as that's the default option these days.  We can do that in a follow up PR though.